spec/abci/abci++_comet_expected_behavior.md

```diff
@@ -224,3 +227,35 @@ As for the new methods:
 Finally, `Commit`, which is kept in ABCI++, no longer returns the `AppHash`. It is now up to
 `FinalizeBlock` to do so. Thus, a slight refactoring of the old `Commit` implementation will be
 needed to move the return of `AppHash` to `FinalizeBlock`.
+
+## Accomodating for vote extensions
+
+In a manner transparent to the application, CometBFT ensures the node is provided with all
+the data it needs to participate in consensus. 
+
+In the case of recovering from a crash, or joining the network via state sync, CometBFT will make
+sure the node acquires the neccessary vote extensions before switching to conensusus. 
+
+If a node is already in consensus but falls behind, during catch-up, CometBFT if providing the node with 
```

Comment:
confusing.
Should `CometBFT if providing` become `CometBFT will provide`?

Reply:
Yes, spelling mistake.